catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -86,32 +109,108 @@ export type FilesAction = tagged.InstanceOf<typeof FilesAction>
 
 export type LocalFile = FileWithPath & FileWithHash
 
+// TODO: queue/array
```

Comment:
what does this comment mean?

Reply:
Use a list of objects instead of a single one. So, we can use infinite "revert"